packages/cli/src/util/env/get-env-records.ts

```diff
@@ -7,6 +7,8 @@ export default async function getEnvRecords(
   output: Output,
   client: Client,
   projectId: string,
+  /** The CLI command that was used that needs the environment variables. */
+  source: string,
```

Comment:
what do you think of using an enum or a union type for what source could be, i.e.

`EnvRecordsSourceType = 'vercel-cli:env:rm' | 'vercel-cli:pull' | ...`
- Will give us autocomplete 
- Will prevent easily avoidable errors like misspelling the source

Reply:
Ideally there's type sharing between `cli` and `api` to prevent typos. I like this idea to prevent typos in `cli`.